source/operations/monitoring.rst

```diff
@@ -28,9 +28,11 @@ collected metrics.
   trigger alerts based on collected metrics. See
   :ref:`minio-metrics-and-alerts-alerting` for more information.
 
-MinIO publishes collected metrics data using Prometheus-compatible data
-structures. Any Prometheus-compatible scraping software can ingest and
-process MinIO metrics for analysis, visualization, and alerting.
+When configured, the :ref:`MinIO Console <minio-console-metrics>` shows some metrics in the :guilabel:`Monitoring > Metrics` page.
```

Comment:
So talking to AB about this, our distinction is this:

- MinIO by default provides point-in-time metrics on cluster status and operations.
- For deployments configured to use Prometheus, the Console can provide time-series data on an additional set of metrics.

Reply:
I reworked this section to incorporate this feedback.